### ISSUE TITLE
EvenLog.getEnterprizeEvents() returns events where boxEvent.getSourceInfo() is always NULL

### ISSUE THREAD

**Issue by D-CURT** [box/box-java-sdk#792]
When I getting enterprise events I cannot obtain a source info.
EventLog.getEnterpriseEvents(
                    adminApi,
                    position,
                    startDate,
                    endDate,
                    BoxEvent.Type.UPLOAD,
                    BoxEvent.Type.EDIT
).forEach(event -> {
        event.getSourceInfo() /* is null */
        event.getSourceJSON() /* it contains something similar like:
        {
	"item_type": "file",
	"item_id": "47069",
	"item_name": "Tetra Pak.pptx",
	"parent": {
		"type": "folder",
		"name": "Tetra Pak",
		"id": "78619"
	},
	"owned_by": {
		"type": "user",
		"id": "394",
		"name": "Jan Brown",
		"login": "janbroun@gmail.com"
	}
        }
        */ 
};

Basically I need to obtain path_collection of each source in events. As I understand it, I can do it via sourceInfo. But this field is always NULL and now it is necessary to perform an additional request to get BoxItem.Info. Are there any alternative possibilities to do that?

I`m using box-java-sdk version - 2.44.1
java version - 11.0.3

Thank you!

**Comment by sujaygarlanka** [comment]
@D-CURT Some events cannot be properly mapped to a BoxResource.Info object, so for these events `event.getSourceInfo()` fails with a null. For these events, the you must use the rawJSON. It is explained [here](https://github.com/box/box-java-sdk/blob/dbc14224ff122f950bab2faa8dfb39fb691356e7/src/main/java/com/box/sdk/BoxEvent.java#L46).

**Comment by D-CURT** [comment]
@sujaygarlanka thank you for the explanation. I saw the description. I just didn`t think that "some enterprise event sources" means all in my case. Sadly but ok. Thank you again.
